tests/test_bs.py

```diff
@@ -48,10 +48,57 @@ def query_bootstrap_es(indx, namespace, bootstrap_po_name):
     return None
 
 
+def setup_bootstrap_in_namespace(namespace, bs_deployment_info, bootstrap_config, oracle=None, poet=None, dep_time_out=120):
+
+    bootstrap_args = {} if 'args' not in bootstrap_config else bootstrap_config['args']
```

Comment:
use `getconf` instead duplicating code

Reply:
This code is different from get_conf.
get_conf is for the clients part.
We can think on how to unify both but this should be the scope of another PR